### ISSUE TITLE
Tagging every issues as a question

### ISSUE THREAD

**Issue by Berkmann18** [ameerthehacker/probot-question-tagger-demo#7]
As far as the bot is meant to tag question issues as a question, it seems to tag those who don't have any questions in them or/and that aren't even meant to be classified as such.
Is this normal?

**Comment by ameerthehacker** [comment]
@Berkmann18 you are right this is not the expected behaviour, this is because the techniques used in this bot for classifying the issues is very rudimentary

**Comment by Berkmann18** [comment]
@ameerthehacker Ah okay. Do you know how to get around that?

**Comment by ameerthehacker** [comment]
Hi @Berkmann18, one better way to get around it is to use a clustering algorithm with classifies similar type questions in a repo into a label. Please take a look at this comment to learn more on it https://github.com/probot/ideas/issues/51#issuecomment-435713514
